pkg/resourceapply/apps_test.go

```diff
@@ -274,8 +274,8 @@ func TestApplyStatefulSet(t *testing.T) {
 			}(),
```

Comment:
Please add test verifying we can override ownerRef containing different Kind in case when UID matches